composer.json

```diff
@@ -23,7 +23,8 @@
     "require": {
         "php": "^7.1",
         "cocur/slugify": "^1.0 || ^2.0 || ^3.0",
-        "doctrine/doctrine-bundle": "^1.0",
+        "doctrine/doctrine-bundle": "^1.0 || ^2.0",
```

Comment:
Sure 1.0 is right? AFAIK it's 1.3

Reply:
Are you talking about doctrine-bundle? It was already1.0 before.
If you're talking about the change for doctrine/persistence it's on the line below and it's 1.3.3 to be specific :)

Reply:
We should use at least doctrine-bundle [1.12.3](https://packagist.org/packages/doctrine/doctrine-bundle#1.12.3). The other bundle did not use the new interface